application/models/Surveymenu.php

```diff
@@ -177,7 +177,15 @@ public function createSurveymenuArray($oSurveyMenuObjects, $collapsed = false, $
                 if ($oDataAttribute->isActive !== null && $oSurvey != null) {
                     if ($oDataAttribute->isActive == true && $oSurvey->active == 'N') {
                         $aEntry['disabled'] = true;
-                        $aEntry['disabled_tooltip'] = sprintf(gT("The '%s' section is not available yet. Please activate your survey to enable this section."), gT($aEntry['menu_title']));
+                        if($aEntry['name']==='responses') {
+
+                            $aEntry['disabled_tooltip'] = sprintf(gT("This survey is not active and has no %s."), gT($aEntry['name']));
```

Comment:
$aEntry['name'] is always "responses" here, no need to use the variable?

Reply:
Also gT($var) should be avoided in general, since the task to pull translations won't understand it.